Multisig/UI/Transaction/ExecuteTransaction/AmountAndValuePiece.swift

```diff
@@ -21,8 +21,19 @@ class AmountAndValuePiece: UINibView {
         setTextAlignment(.right)
     }
 
-    func setAmount(_ value: String?) {
-        amountLabel.text = value
+    func setAmount(_ value: String?, sponsored: Bool = false) {
+        if value != nil && sponsored {
+            amountLabel.attributedText = NSMutableAttributedString(
+                string: value!,
+                attributes: [
+                    NSAttributedString.Key.foregroundColor: UIColor.labelTertiary,
```

Comment:
Please refer to file ConnectKeystoneFlow.swift line 40 and see how we used our custom styles to create attributedString and then append whatever new attributes you want